viite-UI/src/view/RoadAddressProjectEditForm.js

```diff
@@ -3,6 +3,10 @@
     var LinkStatus = LinkValues.LinkStatus;
     var LinkGeomSource = LinkValues.LinkGeomSource;
     var CalibrationCode = LinkValues.CalibrationCode;
+    var projectIncomplete = 1;
```

Comment:
if we have there already defined somewhere like @pedrosag  said, then we should use them instead